src/scripts/librarymenu.js

```diff
@@ -297,6 +297,11 @@ define(["dom", "layoutManager", "inputManager", "connectionManager", "events", "
             icon: "swap_horiz",
             href: "encodingsettings.html",
             pageIds: ["encodingSettingsPage"]
+        }, {
+            name: "Open Subtitles",
```

Comment:
name: globalize.translate("TabOpensubtitles"),

Reply:
add this to strings 
"TabOpensubtitles": "Open Subtitles",

Reply:
Open Subtitles is a name. There is no translation.

Reply:
i agree

Reply:
but to open inside the server coloum the server read the strings

Reply:
I don't really follow. It works as-is. We only need strings for actual translations.

Reply:
my mistake LGTM sorry